frontend/src/summary.pug

```diff
@@ -2,7 +2,7 @@
   form.summary-picker.mui-form--inline(onsubmit="return false;")
     .summary-picker__section
       .mui-textfield.search_box
-        input.summary-picker__search(type="text", v-on:change="updateFilterSearch", v-model="filterSearch")
+        input.summary-picker__search(type="text", v-model="filterSearch")
```

Comment:
note that with this change, the summary charts gets modified while users type in the search. Previously it was only updated when user hits enter. 

Please give a comment on the functionality changes that is not explained in ur PR message (if any) in your future PRs. This makes the review process easier.

Reply:
reset to original implementation